sdk/include/reactos/arc/arc.h

```diff
@@ -554,9 +783,17 @@ typedef struct _LOADER_PARAMETER_BLOCK
     PSTR LoadOptions;
     PNLS_DATA_BLOCK NlsData;
     PARC_DISK_INFORMATION ArcDiskInformation;
+#if (NTDDI_VERSION < NTDDI_WIN8)
     PVOID OemFontFile;
+#endif
+#if (NTDDI_VERSION < NTDDI_WIN7)
     struct _SETUP_LOADER_BLOCK *SetupLdrBlock;
```

Comment:
(Side-note: This one is a possible candidate for a `#ifdef __REACTOS__` at the end of the loader block, since we use it in our freeldr for setup.